From 2f0f1a27bdeab8d73ed3a463454d7d28bd0143a6 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 21 Oct 2019 11:32:09 +0800
Subject: [PATCH] net: rockchip_wlan: rtl8723cs: fix clang compile error

drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c:45:44: error: expansion of date or time macro is not reproducible [-Werror,-Wdate-time]
        RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);

Change-Id: I07e50007edc508ec684eb421cfe5fd4378c96553
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c b/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c
index 1b4298f2422a..4b29808950b4 100644
--- a/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c
+++ b/drivers/net/wireless/rockchip_wlan/rtl8723cs/core/rtw_debug.c
@@ -42,7 +42,6 @@ const char *rtw_log_level_str[] = {
 void dump_drv_version(void *sel)
 {
 	RTW_PRINT_SEL(sel, "%s %s\n", DRV_NAME, DRIVERVERSION);
-	RTW_PRINT_SEL(sel, "build time: %s %s\n", __DATE__, __TIME__);
 }
 
 void dump_drv_cfg(void *sel)
-- 
2.35.3

